.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,78 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Thanks for taking the time to fill out this bug report!
+        ### The following information will help us resolve your issue.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: |
+        Please copy and paste the code you were trying to run that caused the error.
+
+        Feel free to include as little or as much as you think is relevant. This section will be automatically formatted into code, so no need for backticks.
+    validations:
+  - type: textarea
+    id: traceback
+    attributes:
+      label: Traceback
+      description: |
+        Please copy and paste the traceback produced by the error.
+
+        This section will be automatically formatted into code, so no need for backticks.
+      render: python
```

```suggestion
      render: pytb
```

A Python traceback should be rendered as pytb